v2.2/debug-merge-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Merge debug logs for all nodes
+summary: Learn the command for merging the collected debug logs from all nodes in your cluster.
+toc: true
+---
+
+The `debug merge-logs` [command](cockroach-commands.html) merges the logs for all nodes in the cluster collected using the [`debug zip`](https://www.cockroachlabs.com/docs/stable/debug-zip.html) command. This command helps in debugging cluster issues by making it easier to parse the logs for all nodes and locating an issue.
+
+{{site.data.alerts.callout_danger}}
+The file produced by `cockroach debug merge-log` can contain highly sensitive, unanonymized information, such as usernames, passwords, and possibly your table's data. You should share this data only with Cockroach Labs developers and only after determining the most secure method of delivery.
+{{site.data.alerts.end}}
+
+## Subcommands
```

Comment:
Shouldn't this be on a more general article about `cockroach debug` rather than on the `merge-logs` page specifically?

Reply:
I don't think we have a generic debug doc. We do have the `debug zip` and `debug merge-logs` docs, so I put the info on both.